src/app/register/register.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.less']
+})
+export class RegisterComponent implements OnInit {
+
+  constructor() { }
+  model: any = {};
+  canRegister: boolean = true;
```

Comment:
en casos como este no es necesario especificar el tipo de dato ya que es obvio con la inicializacion
especifica el tipo de dato con objetos, donde no es Obvio cual es el contenido
```suggestion
  canRegister = true;
```
para evitar que le pongas `any` a todo usa las interfaces para que tengas auto-completado y prevengas bugs mas adelante
```ts
export interface MyModel {
  email: string;
  nombre?: string; // opcional
  password: string;
}

class RegisterComponent implements OnInit {
  model: MyModel = { email: ''; password: '' };
  canRegister = false;
}
```